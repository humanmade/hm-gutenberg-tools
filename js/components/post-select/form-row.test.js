// jest.mock( 'wp', () => {} );
import * as React from 'react';
// import TestRenderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import FormRow from './form-row';

test( 'Form Row', () => {
// 	const testRenderer = TestRenderer.create(
// 		React.createElement( FormRow, {
// 			label: 'Foo',
// 			labelFor: 'bar',
// 		}, 'test' )
// 	);

// 	console.log( testRenderer.toJSON() );
// 	expect( testRenderer.toJSON() ).toMatchSnapshot();

	const formRow = shallow( <FormRow label="Foo" labelFor="bar">Test</FormRow> );
	console.log( formRow )
	// expect( FormRow.children( '.post-list-item' ).length ).toBe( 0 );
} );
