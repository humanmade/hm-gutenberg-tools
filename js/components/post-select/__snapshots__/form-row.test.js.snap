// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Row 1`] = `
<FormRow
  label="Foo"
  labelFor="bar"
>
  test
</FormRow>
`;
